source/includes/write/insert.c

```diff
@@ -0,0 +1,77 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <bson/bson.h>
+#include <mongoc/mongoc.h>
+
+int
+main(int argc, char *argv[])
+{
+    mongoc_client_t *client;
+    mongoc_collection_t *collection;
+    mongoc_init();
+
+    client =
+        mongoc_client_new ("<connection string URI>");
+    collection = mongoc_client_get_collection (client, "sample_restaurants", "restaurants");
+    
+    {
+        // Inserts a document that stores a "name" value of "Mongo's Burgers" into the collection
+        // start-insert-one
+        bson_t *document = BCON_NEW("name", BCON_UTF8("Mongo's Burgers"));
```

Comment:
C documentation puts a space between the function name and parentheses, so we should do the same.

Applies to this whole file.

```suggestion
        bson_t *document = BCON_NEW ("name", BCON_UTF8("Mongo's Burgers"));
```